pom.xml

```diff
@@ -241,6 +242,12 @@
                 <artifactId>jackson-databind</artifactId>
                 <version>${jacksonDataBindVersion}</version>
             </dependency>
+
+            <dependency>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen</artifactId>
+                <version>${swaggerCondeGenVersion}</version>
```

Comment:
this needed to exclude javax.validation, else it was resulting into https://gist.github.com/kaweesi/82c26267cc799ce8cf183f1475099ec7 which resulted into a great bug at https://issues.openmrs.org/browse/FM-246
Fixed at: https://github.com/openmrs/openmrs-module-fhir/commit/7aad32bdebdde8da3400d35d09172dbb4d4ca556